### PR TITLE
Add overlay for 11TPro

### PR DESCRIPTION
**Pull request by alexandreapis**

### CODE REVIEW

File: Xiaomi/11TPro/res/xml/power_profile.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        <value>400000</value>
+    </array>
+    <array name="cpu.active.cluster0">
+        <value>0.1</value>
+    </array>
+    <item name="cpu.idle">0.1</item>
+    <array name="memory.bandwidths">
+        <value>22.7</value>
+    </array>
+    <item name="battery.capacity">1000</item>
```
Comment (phhusson) [inline Xiaomi/11TPro/res/xml/power_profile.xml:38]: Your device has a 1000mAh battery?

File: Xiaomi/11TPro/res/values/config.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <integer-array name="config_ambientBrighteningThresholds">
+        <item>100</item>
+    </integer-array>
+    <integer-array name="config_ambientDarkeningThresholds">
+        <item>200</item>
+    </integer-array>
+    <array name="config_ambientThresholdLevels" />
+  
+    
+    <array name="config_autoBrightnessButtonBacklightValues" />
```
Comment (phhusson) [inline Xiaomi/11TPro/res/values/config.xml:33]: You actually want to disable automatic brightness control?

File: Xiaomi/11TPro/res/values/config.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="config_useDevInputEventForAudioJack">false</bool>
+    <bool name="config_showNavigationBar">false</bool>
+    <bool name="config_dozeAfterScreenOffByDefault">true</bool>
+    <bool name="config_dozeAlwaysOnDisplayAvailable">true</bool>
+    <bool name="config_powerDecoupleAutoSuspendModeFromDisplay">true</bool>
+    <bool name="config_powerDecoupleInteractiveModeFromDisplay">false</bool>
```
Comment (phhusson) [inline Xiaomi/11TPro/res/values/config.xml:8]: No saying no, but most devices support this to "true" which makes AOD much more power efficient

File: Xiaomi/11TPro/res/values/config.xml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+ 
+   
+    <array name="config_minimumBrightnessCurveNits">
+        <item>0.0</item>
+        <item>50.0</item>
+        <item>90.0</item>
+    </array>
+ 
+
+    <string name="config_mainBuiltInDisplayCutout" />
```
Comment (phhusson) [inline Xiaomi/11TPro/res/values/config.xml:47]: Googling for your device name definitely shows a cutout

**Review by phhusson — CHANGES_REQUESTED** [review]
Pretty sure you didn't test it. The values you got seem completely broken.

**Comment by alexandreapis** [comment]
> Pretty sure you didn't test it. The values you got seem completely broken.


@phhusson yes, I'll test it right now... I'm sorry, I just followed wiki's tutorial and tried to find a pattern between similary devices... but I didnt find a lot of values on describied files...

**Pull request closed without merging**